drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -405,15 +417,23 @@ local function find_default_endpoint(device)
   return device.MATTER_DEFAULT_ENDPOINT
 end
 
-local function assign_child_profile(device, child_ep)
+local function assign_child_profile(device, child_ep, ep_sequence)
   local profile
 
   -- check if device has an overridden child profile that differs from the profile
   -- that would match the child's device type
   for _, fingerprint in ipairs(child_device_profile_overrides) do
     if device.manufacturer_info.vendor_id == fingerprint.vendor_id and
        device.manufacturer_info.product_id == fingerprint.product_id then
-      return fingerprint.child_profile
+      if device.manufacturer_info.vendor_id == AQARA_MANUFACTURER_ID then
+        if ep_sequence == 1 then
+          -- To add Electrical Sensor only to the first EDGE_CHILD
+          device:set_field(IS_AQARA_SWITCH_DEVICE, true)
```

Comment:
```suggestion
          device:set_field(IS_AQARA_SWITCH_DEVICE, true, {persist = true})
```
This persist flag is needed for a note like this

Reply:
Yes, I resolved it.